go/felix/rules/endpoints.go

```diff
@@ -45,6 +46,7 @@ func (r *DefaultRuleRenderer) HostDispatchChains(endpoints map[string]proto.Host
 	for ifaceName, _ := range endpoints {
 		names = append(names, ifaceName)
 	}
+	sort.Strings(names)
```

Comment:
should this be pushed down into `dispatchChains()`?  I bet our test of whether the workload chains have changed is failing right now, making us churn the chains each time.